### PR TITLE
sql: minor RENAME COLUMN fixes in ent integration

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

**Review by yonidavidson — APPROVED** [review]
LGTM

**Review by rotemtam — APPROVED** [review]

File: sql/postgres/migrate.go [hunk 1]
```diff
@@ -195,12 +195,12 @@ func (s *state) modifyTable(ctx context.Context, modify *schema.ModifyTable) err
 			if err != nil {
 				return err
 			}
-			comments = append(comments, s.tableComment(modify.T, to, from))
```
Comment (masseelch) [inline sql/postgres/migrate.go:198]: Why this rename? They are comments, aren't they?

File: sql/postgres/migrate.go [hunk 2]
```diff
@@ -241,17 +241,17 @@ func (s *state) modifyTable(ctx context.Context, modify *schema.ModifyTable) err
 				return err
 			}
 			if c := (schema.Comment{}); sqlx.Has(change.C.Attrs, &c) {
-				comments = append(comments, s.columnComment(modify.T, change.C, c.Text, ""))
```
Comment (masseelch) [inline sql/postgres/migrate.go:244]: Same here

File: sql/postgres/migrate.go [hunk 3]
```diff
@@ -241,17 +241,17 @@ func (s *state) modifyTable(ctx context.Context, modify *schema.ModifyTable) err
...
-			changes = append(changes, change)
+			alter = append(alter, change)
 		case *schema.ModifyColumn:
 			k := change.Change
 			if change.Change.Is(schema.ChangeComment) {
 				from, to, err := commentChange(sqlx.CommentDiff(change.From.Attrs, change.To.Attrs))
 				if err != nil {
 					return err
 				}
-				comments = append(comments, s.columnComment(modify.T, change.To, to, from))
```
Comment (masseelch) [inline sql/postgres/migrate.go:254]: And here

**Review by masseelch — APPROVED** [review]

**Pull request merged**